wiki/Tournaments/o!BAM/1/en.md

```diff
@@ -0,0 +1,576 @@
+
+Sunday, 17 October 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **"5 digit btw"** | **7** | 3 | nomicstreamers | [#1](https://osu.ppy.sh/community/matches/92542468%20https://osu.ppy.sh/community/matches/92543092)
+
+### Finals
+
+Friday, 8 October 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Communist LSD enjoyers** | **7** | 1 | 1v2 | [#1](https://osu.ppy.sh/mp/92085133)
+
+Saturday, 9 October 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| nomicstreamers | 3 | **7** | **"5 digit btw"** | [#1](https://osu.ppy.sh/mp/92140758)
+
+Sunday, 10 October 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| low combo SS | 3 | **7** | **Ascendants** | [#1](https://osu.ppy.sh/community/matches/92202772)
+
+Monday, 11 October 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Communist LSD enjoyers** | **7** | 6 | Ascendants | [#1](https://osu.ppy.sh/mp/92250750)
+
+### Semifinals
+
+Friday, 1 October 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Hutao Enjoyers** | **6** | 2 | ur mom | [#1](https://osu.ppy.sh/mp/91743361)
+| **1v2** | **6** | 2 | Martin n willy show | [#1](https://osu.ppy.sh/mp/91745670)
+| :seemsgood: | 4 | **6** | **Ascendants** | [#1](https://osu.ppy.sh/community/matches/91745843)
+
+Saturday, 2 October 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **nomicstreamers** | **6** | 0 | low combo SS | [#1](https://osu.ppy.sh/mp/91803867)
+| **cutest malaysia couple** | **6** | 0 |	:okniceperson: | [#1](https://osu.ppy.sh/community/matches/91806781)
+
+Sunday, 3 October 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Ascendants** | **6** | 3 | cutest malaysia couple | [#1](https://osu.ppy.sh/community/matches/91865888)
+
+Monday, 4 October 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Hutao Enjoyers | 5 | **6** | **1v2** | [#1](https://osu.ppy.sh/community/matches/91908160)
+| **"5 digit btw"** | **6** | 2 |	Communist LSD enjoyers | [#1](https://osu.ppy.sh/community/matches/91909065)
+
+### Quarterfinals
+
+Friday, 24 September 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **nomicstreamers** | **6** | 1 | 1v2 | [#1](https://osu.ppy.sh/community/matches/91398903)
+| **Martin n willy show** | **6** | 5 | Chicken Sandwich | [#1](https://osu.ppy.sh/community/matches/91402172)
+
+Saturday, 25 September 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **:okniceperson:** | **6** | 1 | Amamiya Kokoro | [#1](https://osu.ppy.sh/community/matches/91443708)
+| **ur mom** | **6** | 1 | stwopid | [#1](https://osu.ppy.sh/community/matches/91450122)
+| **chunchunmaru Mio**** | **6** | 4 | Supremacy | [#1](https://osu.ppy.sh/community/matches/91454313)
```

Comment:
```suggestion
| **chunchunmaru** | **6** | 4 | Supremacy | [#1](https://osu.ppy.sh/community/matches/91454313)
```

Reply:
i really dint save before commiting (not like this)